srcpkgs/isl/template

```diff
@@ -23,8 +24,9 @@ post_install() {
 }
 
 isl-devel_package() {
-	depends="gmp-devel isl>=${version}_${revision}"
+ 	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
+ 	conflicts="isl-devel>=0"
```

Comment:
Why conflict with itself?

Reply:
Thanks for comments both here and in the other pr.
I edited accordingly.